psi4/driver/procrouting/proc.py

```diff
@@ -1017,8 +1017,12 @@ def scf_wavefunction_factory(name, ref_wfn, reference):
         raise ValidationError("SCF: Unknown reference (%s) when building the Wavefunction." % reference)
 
     if disp_type:
-        wfn._disp_functor = empirical_dispersion.EmpericalDispersion(
-            disp_type[0], disp_type[1], tuple_params=modified_disp_params)
+        if len(disp_type) > 2:
```

Comment:
If possible, it might be nice to be able to check this off keys rather than length.